# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/VideoDecoder.h
# Commit: 02cb31bfdc0f
# Full Hash: 02cb31bfdc0f4a15397b3a4e0725c408d738b34c
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-06-30 04:06:43
# Regressor Bug: 1831451
# File Overlap Count: 1
# Description:
#   Bug 1831451 - Implement VideoDecoder::IsConfigSupported r=padenot
#   
#   This patch implements the isConfigSupported method for WebCodecs'
#   VideoDecoder.
#   
# ==============================================================================

diff -r 77b0f5226ed8 -r 02cb31bfdc0f dom/media/webcodecs/VideoDecoder.h
--- a/dom/media/webcodecs/VideoDecoder.h	Thu Jun 29 22:33:11 2023 +0000
+++ b/dom/media/webcodecs/VideoDecoder.h	Thu Jun 29 22:33:11 2023 +0000
@@ -71,7 +71,7 @@
   void Close(ErrorResult& aRv);
 
   static already_AddRefed<Promise> IsConfigSupported(
-      const GlobalObject& global, const VideoDecoderConfig& config,
+      const GlobalObject& aGlobal, const VideoDecoderConfig& aConfig,
       ErrorResult& aRv);
 };
 